setup.cfg

```
@@ -20,7 +20,7 @@ install_requires =
     astropy>=4.0
     glue-core>=1.6.1
     regions>=0.4
-    specutils>=0.7
+    specutils>=2.0
```

This would let the install happen but would also cause this PR to not able to be merged because PyPI disallows dev pins for releases.

```suggestion
    specutils>=2.0.dev
```